js/renovation/ui/toolbar/toolbar.tsx

```diff
@@ -8,18 +8,31 @@ import LegacyToolbar from '../../../ui/toolbar';
 
 import { DomComponentWrapper } from '../common/dom_component_wrapper';
 import { ToolbarProps } from './toolbar_props';
+import { isDefined } from '../../../core/utils/type';
 
 export const viewFunction = ({
   props,
   restAttributes,
-}: Toolbar): JSX.Element => (
-  <DomComponentWrapper
-    componentType={LegacyToolbar}
-    componentProps={props}
-  // eslint-disable-next-line react/jsx-props-no-spreading
-    {...restAttributes}
-  />
-);
+}: Toolbar): JSX.Element => {
+  const { rtlEnabled, items } = props;
+
+  items?.forEach((item) => {
```

Comment:
Use map it's immutable and  allow you to remove /* eslint-disable no-param-reassign */

Reply:
Yep, fixed via creating clone and traversing items via for cycle